src/sentry/api/event_search.py

```diff
@@ -2164,6 +2164,22 @@ def result_type_fn(function_arguments, parameter_values):
             default_result_type="duration",
             redundant_grouping=True,
         ),
+        Function(
+            "var",
+            required_args=[NumericColumnNoLookup("column")],
+            aggregate=["varSamp", ArgValue("column"), None],
```

Comment:
@Zylphrex could this return infinity? As per https://clickhouse.tech/docs/en/sql-reference/aggregate-functions/reference/varsamp/

Reply:
yeah, I noticed that varSampStable returns inf but varSamp return NaN

Reply:
I think adding `-OrNull` combinator might suffice. I'm unsure if it covers the inf case.

Reply:
I think it's just because of the stability of the function. Basically, the breaking case is when n=1, since that breaks the sampling formula. if we used the population formulas this wouldn't be a problem, but I think could get very computationally intensive